roles/jenkins_builder/tasks/main.yml

```diff
@@ -63,6 +63,7 @@
   - /d/build
   - /d/logs
   - /home/jenkins/root
+  - /home/jenkins/root/workspace
```

Comment:
This seems unrelated to bugziller deployment, what prompted this ?

Reply:
Without this folder being precreated the setfacl command in the next step will fail on fresh nodes

Reply:
Indeed, and it seems to be related to the failure we see on https://bugzilla.redhat.com/show_bug.cgi?id=1488125 so I cherry picked that fix.